python/docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
RMM doesn't support Windows, do we need to support building docs on Windows?

Reply:
rm'd